pkgs/development/libraries/libgphoto2/default.nix

```diff
@@ -58,10 +58,13 @@ stdenv.mkDerivation rec {
           buildPackages.libgphoto2;
     in
     ''
-      mkdir -p $out/lib/udev/rules.d
+      mkdir -p $out/lib/udev/{rules.d,hwdb.d}
       ${executablePrefix}/lib/libgphoto2/print-camera-list \
-          udev-rules version 175 group camera \
-          >$out/lib/udev/rules.d/40-gphoto2.rules
+          udev-rules version 201 group camera \
+          >$out/lib/udev/rules.d/40-libgphoto2.rules
+      ${executablePrefix}/lib/libgphoto2/print-camera-list \
+          hwdb version 201 group camera \
+          >$out/lib/udev/hwdb.d/20-gphoto.conf
```

Comment:
Actually, both Fedora and arch use hwdb as file extension, not sure if that is important.

Reply:
https://man.archlinux.org/man/core/systemd/hwdb.7.en
> . hwdb files must have the extension .hwdb; other extensions are ignored.

Reply:
Would be nice to fix that upstream. Or even better, make the build system install these files. But autotools are annoying.